### PR TITLE
fix(sidebar): Fixed sidebar hovers on headers.

### PR DESCRIPTION
**Pull request by michaelkleinhenz**
Removed the hover behaviour on headers in the sidebar. Also fixes the extra "123" on the All header.

### CODE REVIEW

**Comment by fabric8cd** [comment]
@michaelkleinhenz fabric8/fabric8-ui:SNAPSHOT-PR-2286-1 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2286-fabric8-planner.badger.fabric8.io

File: src/app/components/side-panel/side-panel.component.html [hunk 1]
```diff
@@ -13,7 +13,7 @@
       placement="right"></span>
     <h3 class="margin-0"
       [class.show]="sidePanelOpen"
-      [class.hide]="!sidePanelOpen">All 123</h3>
+      [class.hide]="!sidePanelOpen">All</h3>
 </a>
```
Comment (nimishamukherjee) [inline src/app/components/side-panel/side-panel.component.html]: This text has been corrected and is in master

**Comment by fabric8cd** [comment]
@michaelkleinhenz fabric8/fabric8-ui:SNAPSHOT-PR-2286-2 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2286-fabric8-planner.badger.fabric8.io

**Comment by michaelkleinhenz** [comment]
Note: this branch currently fails on cd due to some issues with the chromedriver. Locally, the tests run fine.

**Comment by fabric8cd** [comment]
@michaelkleinhenz fabric8/fabric8-ui:SNAPSHOT-PR-2286-6 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2286-fabric8-planner.badger.fabric8.io

**Comment by fabric8cd** [comment]
@michaelkleinhenz fabric8/fabric8-ui:SNAPSHOT-PR-2286-10 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2286-fabric8-planner.badger.fabric8.io

**Comment by fabric8cd** [comment]
@michaelkleinhenz fabric8/fabric8-ui:SNAPSHOT-PR-2286-11 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2286-fabric8-planner.badger.fabric8.io

**Pull request merged**